main.go

```diff
@@ -517,7 +518,9 @@ Features detected:
 		fmt.Printf("⚠️ Pro gateway detected, but autoscaler is not enabled\n")
 	}
 
-	for namespace, functions := range nsFunctions {
+	for _, namespace := range functionNamespaces {
+		functions := nsFunctions[namespace]
```

Comment:
Best check if this was "OK" for if the namespace is empty

(I forget the order of ok/value), but: 
if ok, value := map[key] ; ok {

}

Reply:
I added this check.